challenges/objects-arrays.js

```diff
@@ -98,28 +146,41 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+
+zooAnimals.forEach( (animal,i) => lowerCase.push(zooAnimals[i].animal_name.toLowerCase()));
+
+console.log('\n The zoo animals\' names all lowercased: ');
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
-console.log(largerPopulation);
+
+const smallerPopulation = zooAnimals.filter((animal,i) => zooAnimals[i].population < 5);
+const smallerNames = smallerPopulation.map( (animal,i) => (`${smallerPopulation[i].animal_name}, Population: ${smallerPopulation[i].population}`));
```

Comment:
It's awesome that you used .map() here for better formatting! Know that you don't need to create a separate variable here, as you can just chain the .map() method on after your .filter() method.